### DISCUSSION TITLE
EAN-13 Barcode issue.

### DISCUSSION THREAD

**Discussion by NorAfguriye** [Blair2004/NexoPOS#465] [Q&A] [answered]
Hi, I am struggling to the EAN-13 or all the barcodes.  i entered 13 numbers but once i tried to save it throws error on the below snapshot. 

tried several changes but still error couldn't pass please support

![image](https://user-images.githubusercontent.com/89262228/148033920-66f1c659-81f2-44a9-a7e7-187b5ed3be5f.png)

**Comment by Blair2004** [comment, marked as answer]
Hi, Use code128 for now.
